specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2018-12-20/examples/AzureIaasVm/GetAccessToken_Get.json

```diff
@@ -40,7 +43,14 @@
           "coordinatorServiceStampUri": "https://sea-bvtd2-coord1-t56tl.ext.trafficmanager.net",
           "protectionServiceStampId": "253c2223-fefd-483d-912f-55fb46ce0eab",
           "protectionServiceStampUri": "https://sea-bvtd2-prot1-t56tl.ext.trafficmanager.net",
-          "tokenExtendedInformation": "<IaaSVMRecoveryPointMetadataBase xmlns:i=\"http://www.w3.org/2001/XMLSchema-instance\" i:type=\"IaaSVMRecoveryPointMetadata_V2015_09\" xmlns=\"http://windowscloudbackup.com/CloudCommon/V2011_09\"><MetadataVersion>V2015_09</MetadataVersion><ContainerType i:nil=\"true\" /><InstantRpGCId>1a2311d9-66f5-47d3-a9fb-7a37da63934b;BackUpOperationJobId;e1ebcf24-81a5-434e-b64b-cc81be65ae16</InstantRpGCId><IsBlockBlobEnabled>false</IsBlockBlobEnabled><IsManagedVirtualMachine>false</IsManagedVirtualMachine><OriginalSAOption>true</OriginalSAOption><OsType>Windows</OsType><ReadMetadaFromConfigBlob i:nil=\"true\" /><RecoveryPointConsistencyType>CrashConsistent</RecoveryPointConsistencyType><RpDiskDetails i:nil=\"true\" /><SourceIaaSVMRPKeyAndSecret i:nil=\"true\" /><SourceIaaSVMStorageType>NormalStorage</SourceIaaSVMStorageType><VMSizeDescription>Standard_D2s_v3</VMSizeDescription></IaaSVMRecoveryPointMetadataBase>"
+          "tokenExtendedInformation": "<IaaSVMRecoveryPointMetadataBase xmlns:i=\"http://www.w3.org/2001/XMLSchema-instance\" i:type=\"IaaSVMRecoveryPointMetadata_V2015_09\" xmlns=\"http://windowscloudbackup.com/CloudCommon/V2011_09\"><MetadataVersion>V2015_09</MetadataVersion><ContainerType i:nil=\"true\" /><InstantRpGCId>1a2311d9-66f5-47d3-a9fb-7a37da63934b;BackUpOperationJobId;e1ebcf24-81a5-434e-b64b-cc81be65ae16</InstantRpGCId><IsBlockBlobEnabled>false</IsBlockBlobEnabled><IsManagedVirtualMachine>false</IsManagedVirtualMachine><OriginalSAOption>true</OriginalSAOption><OsType>Windows</OsType><ReadMetadaFromConfigBlob i:nil=\"true\" /><RecoveryPointConsistencyType>CrashConsistent</RecoveryPointConsistencyType><RpDiskDetails i:nil=\"true\" /><SourceIaaSVMRPKeyAndSecret i:nil=\"true\" /><SourceIaaSVMStorageType>NormalStorage</SourceIaaSVMStorageType><VMSizeDescription>Standard_D2s_v3</VMSizeDescription></IaaSVMRecoveryPointMetadataBase>",
+          "identityInfo": {
+            "isSystemAssignedIdentity": true,
+            "managedIdentityId": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/DefaultResourceGroup-WEU/providers/Microsoft.RecoveryServices/vaults/vault998",
+            "managedIdentityUrl": "https://control-southeastasia.identity.azure.net/subscriptions/f2edfd5d-5496-4683-b94f-b3588c579009/resourcegroups/DefaultResourceGroup-WEU/providers/Microsoft.RecoveryServices/vaults/vault998/credentials/v2/systemassigned?arpid=6031f346-6e23-4afe-a3f1-d8067150d5dc&said=31883776-6cdc-46d1-970a-535ef6b8ff06&tid=72f988bf-86f1-41af-91ab-2d7cd011db47",
```

Comment:
These URLs are internal and not exposed to customers.  If you feel there is some scenario that requires it to be exposed please get explicit approval from the managed identity team (https://armwiki.azurewebsites.net/authorization/managed_identities/MSIOnboardingContactUs.html)